bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -227,36 +189,39 @@ private static Icon bagsIcon(boolean enabled) {
             private final Option<GuideGroup> group;
             private final List<Row> children;
 
-            GroupRow(boolean enabled, int index, GuideGroup group, List<Row> children) {
-                super(enabled, group.getName().getOrElse("Guide Group " + index), "", None.<SPTarget>instance(), None.<Long>instance());
+            GroupRow(final boolean enabled, final int index, final GuideGroup group, final List<Row> children) {
+                super(enabled, group.getName().getOrElse("Guide Group " + index), "", None.instance(), None.instance());
                 this.group    = new Some<>(group);
                 this.children = Collections.unmodifiableList(children);
             }
 
             @Override public Option<GuideGroup> group() { return group; }
-            @Override public List<Row> children() { return children; }
+            @Override public List<Row> children()       { return children; }
+            @Override public Icon getIcon()             { return groupIcon; }
         }
 
         private final ImList<Row> rows;
         private final ImList<Magnitude.Band> bands;
         private final ImList<String> columnHeaders;
         private final TargetEnvironment env;
-        private final JXTreeTable treeTable;
 
         TableData() {
             rows          = DefaultImList.create();
             bands         = DefaultImList.create();
             columnHeaders = computeColumnHeaders(bands);
             env           = null;
-            treeTable     = null;
         }
 
-        private static Option<AgsGuideQuality> guideQuality(Option<Tuple2<ObsContext, AgsMagnitude.MagnitudeTable>> ags, final GuideProbe guideProbe, final SPTarget guideStar) {
+        private static Option<AgsGuideQuality> guideQuality(final Option<Tuple2<ObsContext, AgsMagnitude.MagnitudeTable>> ags,
+                                                            final GuideProbe guideProbe, final SPTarget guideStar) {
             return ags.flatMap(tup -> {
                 if (guideProbe instanceof ValidatableGuideProbe) {
+                    final ObsContext ctx = tup._1();
+                    final AgsMagnitude.MagnitudeTable magTable = tup._2();
                     final ValidatableGuideProbe vgp = (ValidatableGuideProbe) guideProbe;
-                    return AgsRegistrar.instance().currentStrategyForJava(tup._1()).map(strategy -> {
-                        final Option<AgsAnalysis> agsAnalysis = strategy.analyzeForJava(tup._1(), tup._2(), vgp, ModelConverters.toSideralTarget(guideStar));
+                    return AgsRegistrar.instance().currentStrategyForJava(ctx).map(strategy -> {
```

Comment:
Would this ever be called for GEMS? I ask because we still have 2 ways to call Gems AGS, the one you are using should be preferred but in many places we are still using the legacy one